Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.Tests.ps1

```diff
@@ -87,11 +132,23 @@ Describe “Add-IshBackgroundTask" -Tags "Create" {
 			{$ishBackgroundTaskParameters = Add-IshBackgroundTask -EventType $ishEventTypeToPurge -EventDescription $eventDescription -RawInputData $null} | Should Throw
 		}
 	}
+	Context "Add-IshBackgroundTask (Parameters group with StartAfter)" {
+		$rawData = "<data><dataExample>Text</dataExample></data>"
+		$eventDescription = "Created by Powershell and ISHRemote (startAfter)"
+		$dateTomorrow = (Get-Date).AddDays(1)
+		$ishBackgroundTaskParameters = Add-IshBackgroundTask -EventType $ishEventTypeToPurge -EventDescription $eventDescription -RawInputData $rawData -StartAfter $dateTomorrow
+		It "Verify object properties returned by Add-IshBackgroundTask" {
+			$ishBackgroundTaskParameters.Count | Should BeExactly 1
+			$ishBackgroundTaskParameters.GetType() | Should BeExactly Trisoft.ISHRemote.Objects.Public.IshBackgroundTask
+			$ishBackgroundTaskParameters.EventType | Should BeExactly $ishEventTypeToPurge
+			$ishBackgroundTaskParameters.userid | Should BeExactly $ishSession.IshUserName
```

Comment:
Can you also check the StartAfter?

Reply:
Ok, we can implement check on the passed StartDate and retrieved StartDate with the assumption that a certain culture for datetime on the server is the constant across all Docs environments